aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -132,10 +132,27 @@
                     echo $timestamp_CPU, $ram_used_available
             done
       path: /home/ubuntu/write-cpu-ram-usage.sh
-
+    - content: |
+            #!/bin/bash
+            pip install setuptools
+            pip install jupyter
+      path: /home/ubuntu/install-jupyter.sh
```

Comment:
Can we just run the same docker image without installing jupyter?

Reply:
Otherwise, if using of docker image is impossible, we might install jupyter on instance and made a new beam AWS image to avoid installing jupyter on it every time. 
Personally I do not like installing things in lambda - it might suddenly break in future.

Reply:
Another argument against installing jupyter and to use docker - we might enhance docker with required python packages and re-use that image everywhere, otherwise it will be required to configure instance every time, or, change lambda with additional installationa packages, which might be a pain.